tests/optimizers/botorch_base_test.py

```diff
@@ -155,3 +156,28 @@ def test_find_optimum_in_1d_discrete_space(seed):
     assert (
         sum(l == 0 for l in losses) > 5
     ), "After figuring out the best of the three points, it should only propose that."
+
+
+def test_get_numerical_points_from_discrete_space():
+    p0l, p0h = -5, 10
```

Comment:
`p0l` and `p0h` seem to be unnecessarily difficult to understand at first glance, since it seems to me we wouldn't loose anything by instead going with `p0_bounds = (-5, 10)`, what do you think?

Reply:
There might be something even better, let's ignore this proposal.